concrete/src/Form/Service/Form.php

```diff
@@ -435,10 +435,11 @@ public function search($key, $valueOrMiscFields = '', $miscFields = [])
      */
     public function __call($name, $args)
     {
-        $key = $args[0];
-        $valueOrMiscFields = $args[1];
-        $miscFields = array_slice($args,2);
-        return $this->inputType($key, $name, $valueOrMiscFields, $miscFields);
+        if(is_array($args[2])){
```

Comment:
Does `$args` *ever* contain at least 3 items? If not, accessing `$args[2]` would throw a warning.

I'd write instead:
```php
if (is_array($args[2] ?? null)) {
```